packages/nimble-components/src/table-column/enum-base/cell-view/template.ts

```diff
@@ -0,0 +1,9 @@
+import { html, repeat } from '@microsoft/fast-element';
+
+import type { TableColumnEnumCellView } from '.';
+import type { Mapping } from '../../../mapping/base';
+
+// prettier-ignore
+export const template = html<TableColumnEnumCellView>`
+    ${repeat(x => [x.getMappingToRender()], html<Mapping>`${x => x?.cellViewTemplate}`)}
```

Comment:
I believe this was the only way to get an instance of the `ConvertedKeyMapping` to render. What is a bit confusing to me is the mismatch in types. Generally, the typing for the inner `html` block within the `repeat` directive is typed to the element type that is being iterated over by the `repeat` (which is `ConvertedKeyMapping | null` here. Yet the type given to the `html` directive is `Mapping`. I presume this only works because the types align on the `cellViewTemplate` member? Is there a reason this isn't:
```html
${repeat(x => [x.getMappingToRender()], html<ConvertedKeyMapping>`${x => x?.cellViewTemplate}`)}
```

which, if that is valid, then I think Molly is right in that this could become:
```
html<TableColumnEnumCellView>`
    ${x => x.getMappingToRender()?.cellViewTemplate}
`;
```

Reply:
Yes, previously, `getMappingToRender()` actually returned a `Mapping`, but I missed this spot when I changed it to return `ConvertedKeyMapping`.

While doing some other refactoring, I made the change you guys pointed out. The only drawback was that I could no longer have template bindings to members of `ConvertedKeyMapping`, e.g. 
```
html<ConvertedKeyMapping>`
    <nimble-spinner title=${x => x.label} ...>
`
```
Because now the `x` in my template is an instance of a cell view or group header view--not a `ConvertedKeyMapping`. Because the `ConvertedKeyMapping` is the one providing the cell view (or group header view) template, I was able to just use `this.label` in my template instead of `x => x.label`. It's okay to do that, because I already have notifiers watching for changes to the `Mapping`s and recreating the templates whenver there's a change.

Also, FYI, I renamed `ConvertedKeyMapping` to `MappingConfig` (and related `MappingConfigIcon`, etc).